x-pack/plugins/beats_management/public/pages/main/beats_action_area.tsx

```diff
@@ -55,108 +60,111 @@ export class BeatsActionArea extends React.Component<any, any> {
     this.pinging = false
   }
   public render() {
+
     const {
       match,
-      history,
+      goTo,
       libs,
-    } = this.props
+    } = this.props;
   return (
-  <div>
-    <EuiButtonEmpty
-      onClick={() => {
-        window.alert('This will later go to more general beats install instructions.');
-        window.location.href = '#/home/tutorial/dockerMetrics';
-      }}
-    >
-      Learn how to install beats
-    </EuiButtonEmpty>
-    <EuiButton
-      size="s"
-      color="primary"
-      onClick={async () => {
-        const token = await libs.tokens.createEnrollmentToken();
-        history.push(`/overview/beats/enroll/${token}`);
-        this.waitForToken(token);
-      }}
-    >
-      Enroll Beats
-    </EuiButton>
+    <div>
+      <EuiButtonEmpty
+        onClick={() => {
+          // random, but spacific number ensures new tab does not overwrite another _newtab in chrome
```

Comment:
hahah mash the keyboard. "random" in that its not 3000 or something another dev of another app might have used. Specific in that its hard coded and not randomly generated. Ion this way if a user clicks this link 20 times its not going to open 20 tabs. Also it will not overwrite another tab or the URL changed by another app opening something in a new tab if someone leaves the tab open

Reply:
Ah sorry - I should have been clearer, I just meant there's a typo in the comment, "specific" was misspelled 👼